Makefile

```diff
@@ -772,6 +772,8 @@ deploy/service: ALLOW_DEVELOPER_INSTANCE ?= "true"
 deploy/service: QUOTA_TYPE ?= "quota-management-list"
 deploy/service: STRIMZI_OLM_INDEX_IMAGE ?= "quay.io/osd-addons/managed-kafka:production-82b42db"
 deploy/service: KAS_FLEETSHARD_OLM_INDEX_IMAGE ?= "quay.io/osd-addons/kas-fleetshard-operator:production-82b42db"
+deploy/service: OBSERVABILITY_OPERATOR_INDEX_IMAGE ?= "quay.io/rhoas/observability-operator-index:v3.0.14"
+deploy/service: OBSERVABILITY_OPERATOR_STARTING_CSV ?= "observability-operator.v3.0.14"
```

Comment:
Can we update this doc section as well https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/4d81751ebd17a92ad43a918106b37e58ba6d8e05/docs/deploying-kas-fleet-manager-to-openshift.md#deploy-kas-fleet-manager?